src/libraries/System.Private.CoreLib/src/System/Text/NewLineUtility.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text
+{
+    internal static class NewLineUtility
```

Comment:
>NewLineUtility [](http://example.com/codeflow?start=26&length=14)

why we call it New Line? why not just Line? just curious no real objection :-)

Reply:
Deleted this file and moved it into string.cs as an internal helper. Also updated the comments to more clearly reflect that the Unicode Standard refers to these as "newline chars" and "newline functions". Hopefully this reads a bit more cleanly now!